provider/datagen/src/cldr/transform/datetime/symbols.rs

```diff
@@ -53,6 +53,12 @@ fn get_era_code_map(calendar: &str) -> BTreeMap<String, TinyStr16> {
         ]
         .into_iter()
         .collect(),
+        "indian" => vec![
```

Comment:
Not sure if this is right

Reply:
It should map `0` to `saka`

Reply:
actually it should also map `"bc"` to `"saka"`, i don't know why the data has that in it